From 0ddb497a31597ac82897f8c08555cdf2c66e09c6 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:35:37 +0100
Subject: [PATCH 5/5] Squash of
 github/lineage-16.0(7fd7d37ad)..lineage-16.0-android-9.0.0_r31(eceab1f01) (DO
 NOT SUBMIT)

Adam Seaton (10):
 - [DO NOT MERGE] Update platform security string to 2018-11-01 on
   oc-mr1-dev Bug:114110585
 - [DO NOT MERGE] Update platform security string to 2018-11-05 on
   oc-mr1-dev bug:114110585
 - [automerger skipped] [DO NOT MERGE] Update platform security string
   to 2018-11-01 on oc-mr1-dev Bug:114110585 am: b703503c1d  -s ours
 - [automerger skipped] [DO NOT MERGE] Update platform security string
   to 2018-11-05 on oc-mr1-dev bug:114110585 am: 281cb3b172  -s ours
 - [DO NOT MERGE] Update platform security String to 2018-12-01
   bug:117667960
 - [DO NOT MERGE] Update platform security string to 2018-12-05 bug:
   117667960
 - [DO NOT MERGE] Update Security String to 2018-12-01 for December
   Release bug:117667960
 - [DO NOT MERGE]Update Security String to 2018-12-05 for December
   release bug: 117667960
 - [automerger skipped] [DO NOT MERGE] Update platform security String
   to 2018-12-01 bug:117667960 am: b43ff9d8f5  -s ours
 - [automerger skipped] [DO NOT MERGE] Update platform security string
   to 2018-12-05 bug: 117667960 am: 7af38a13f3  -s ours

Android Build Merger (Role) (10):
 - [automerger] [RESTRICT AUTOMERGE] Remove build/libs am: 62a11cf698
 - [automerger] [RESTRICT AUTOMERGE] Remove build/libs am: 62a11cf698
   am: 41a932d505
 - [automerger] [RESTRICT AUTOMERGE] Remove build/libs am: 62a11cf698
   am: 41a932d505 am: 2df512843f
 - [automerger] [RESTRICT AUTOMERGE] Remove build/libs am: 62a11cf698
   am: 41a932d505 am: 2df512843f am: f74efccc8d
 - [automerger] [RESTRICT AUTOMERGE] Remove build/libs am: 62a11cf698
   am: 41a932d505 am: 2df512843f am: f74efccc8d skipped: 0c824a631e
 - [automerger] [RESTRICT AUTOMERGE] Add new Android.mk to handle repo
   move am: fd72eaab91
 - [automerger] [RESTRICT AUTOMERGE] Add new Android.mk to handle repo
   move am: fd72eaab91 am: 69d914c57b
 - [automerger] [RESTRICT AUTOMERGE] Add new Android.mk to handle repo
   move am: fd72eaab91 am: 69d914c57b am: 5c6df3c9a2
 - [automerger] [RESTRICT AUTOMERGE] Add new Android.mk to handle repo
   move am: fd72eaab91 am: 69d914c57b am: 5c6df3c9a2 am: cd5dfa01ed
 - [automerger] [RESTRICT AUTOMERGE] Add new Android.mk to handle repo
   move am: fd72eaab91 am: 69d914c57b am: 5c6df3c9a2 am: cd5dfa01ed
   skipped: 428866c804

Elisa Pascual Trevino (5):
 - Update platform security string to 2018-11-01 on pi-dev & master
   Bug:114110585
 - Update platform security string to 2018-11-05 on pi-dev & master
   Bug:114110585
 - Merge changes Id72175ed,I5f83105c into pi-dev
 - [DO NOT MERGE] Security String update to 2019-02-01 Bug: 120144419
   (cherry picked from commit 14b4156d8d9c8269a714096fbb85e15cb2762581)
 - [DO NOT MERGE] Security String update to 2019-02-05 Bug: 120144419
   (cherry picked from commit a9c8e6165e201ba44c6fb485d4e73c43cb32175d)

JP Sugarbroad (9):
 - [RESTRICT AUTOMERGE] Remove build/libs
 - [RESTRICT AUTOMERGE] Add new Android.mk to handle repo move
 - Merge changes from topic "am-b8fbc025-dd1b-4d0d-90bb-1a3f2f2e8d15"
   into oc-dev
 - [automerger] [RESTRICT AUTOMERGE] Add new Android.mk to handle repo
   move am: fd72eaab91 am: 69d914c57b am: 5c6df3c9a2 am: cd5dfa01ed
   skipped: 428866c804 am: 58265f83b2
 - Merge changes from topic "am-b8fbc025-dd1b-4d0d-90bb-1a3f2f2e8d15"
   into oc-dev am: 3266dbabf9
 - [automerger] [RESTRICT AUTOMERGE] Add new Android.mk to handle repo
   move am: fd72eaab91 am: 69d914c57b am: 5c6df3c9a2 am: cd5dfa01ed
   skipped: 428866c804 am: 58265f83b2 am: 422182a1ac
 - Merge changes from topic "am-b8fbc025-dd1b-4d0d-90bb-1a3f2f2e8d15"
   into oc-dev am: 3266dbabf9 am: 073a48b27d
 - [automerger skipped] [automerger] [RESTRICT AUTOMERGE] Add new
   Android.mk to handle repo move am: fd72eaab91 am: 69d914c57b am:
   5c6df3c9a2 am: cd5dfa01ed skipped: 428866c804 am: 58265f83b2 am:
   422182a1ac am: 839e8e5621  -s ours
 - Merge changes from topic "am-b8fbc025-dd1b-4d0d-90bb-1a3f2f2e8d15"
   into oc-dev am: 3266dbabf9 am: 073a48b27d am: aebb4e4c60

Lubo Hristov (2):
 - [DO NOT MERGE] Security string update to 2019-01-01 bug:118903919
 - [DO NOT MERGE] Security string update to 2019-01-05 bug:118903919

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

TreeHugger Robot (1):
 - Merge "[DO NOT MERGE]Update Security String to 2018-12-05 for
   December release bug: 117667960" into pi-dev

android-build-team Robot (22):
 - Make change and version bump to PQ2A.181002.001
 - Make change and version bump to PQ2A.181002.002
 - Make change and version bump to PQ2A.181005.001
 - Make change and version bump to PQ2A.181005.001
 - Merge "Make change and version bump to PQ2A.181005.001" into
   pi-qpr2-release
 - Make change and version bump to PQ2A.181009.001
 - Make change and version bump to PQ2A.181012.001
 - Make change and version bump to PQ2A.181016.001
 - Version bump to PQ2A.181016.002
 - Make change and version bump to PQ2A.181023.001
 - Make change and version bump to PQ2A.181030.001
 - Make change and version bump to PQ2A.181106.001
 - Make change and version bump to PQ2A.181107.001
 - Make change and version bump to PQ2A.181113.001
 - Make change and version bump to PQ2A.181115.001
 - Version bump to PQ2A.181115.002
 - Make change and version bump to PQ2A.181120.001
 - Make change and version bump to PQ2A.181115.003
 - Version bump to PQ2A.181115.004
 - Version bump to PQ2A.181115.005
 - Make change and version bump to PQ2A.190201.001
 - Make change and version bump to PQ2A.190205.001

Change-Id: I1e068717364ed7a7a91d4f0cd2b8080c835fd270

Change-Id: Iea852c79a4707155555bc6b61a906b28441e1cd4
---
 core/build_id.mk         | 2 +-
 core/version_defaults.mk | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index 6823379dd..a1a64e32e 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=PQ1A.190105.004
+export BUILD_ID=PQ2A.190205.001
diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 84e856f69..57e5492f9 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -231,7 +231,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2019-01-05
+      PLATFORM_SECURITY_PATCH := 2019-02-05
 endif
 
 ifndef PLATFORM_SECURITY_PATCH_TIMESTAMP
-- 
2.17.1

